scripts/exrrfs_jedivar.sh

```diff
@@ -144,6 +148,18 @@ if [[ ${start_type} == "warm" ]] || [[ ${start_type} == "cold" && ${COLDSTART_CY
   # check the status
   export err=$?
   err_chk
+  if [[ ${start_type} == "warm" ]] && [[ -v SNUDGETYPES ]]; then
+      # pyioda libraries
+      PYIODALIB=$(echo "$HOMErdasapp"/build/lib/python3.*)
+      export PYTHONPATH="${PYIODALIB}:${PYTHONPATH}"
+      python ${USHrrfs}/snudge.py ${CDATE} ${SNUDGETYPES} ${DATA}/${initial_file} 
```

Comment:
Line 155: NCO would like every USH scripts can run directly from the command line without the leading `python` or `bash` commands.

Could you make `snudge.py` executable, add the following shebang to the top of this file and then remove `python` from line 155?
```
#!/usr/bin/env python
```